internal/golangcilint/golangci_lint.go

```diff
@@ -18,190 +17,197 @@ import (
 )
 
 var configTmpl = template.Must(template.New("golangci").Parse(strings.TrimSpace(strings.ReplaceAll(`
+version: "2"
 run:
-	timeout: {{ .Timeout }} # 1m by default
 	modules-download-mode: {{ .ModDownloadMode }}
```

Comment:
This line has tab indentation, but everything else has space indentation now. This should be consistent. I would prefer tab indentation since the mix of spaces in `configTmpl` vs. tabs in Go code confuses my editor's indentation detection.